pkgs/os-specific/linux/bionic-prebuilt/default.nix

```diff
@@ -4,23 +4,35 @@
 }:
 let
 
-  prebuilt_crt = fetchzip {
+  prebuilt_crt = if stdenv.targetPlatform.isAarch64 then (fetchzip {
     url =  "https://android.googlesource.com/platform/prebuilts/gcc/linux-x86/aarch64/aarch64-linux-android-4.9/+archive/98dce673ad97a9640c5d90bbb1c718e75c21e071/lib/gcc/aarch64-linux-android/4.9.x.tar.gz";
     sha256 = "sha256-LLD2OJi78sNN5NulOsJZl7Ei4F1EUYItGG6eUsKWULc=";
     stripRoot = false;
-  };
+  }) else (fetchzip {
```

Comment:
Else if x86?

Reply:
I guess we could add x86 support too. I didn't have time to track down the prebuilts for x86 and add them though. Perhaps in a separate PR?

Reply:
Oops, that's not what I meant. I meant, we don't want to unconditionally do an `else`, but rather `else if x86_64`. Then it's clear that other types are unsupported.

Reply:
Ok - that makes sense. I'll make that change

Reply:
This is resolved by moving to the 'switch' idiom.